.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: "Setup Java"
         uses: actions/setup-java@v1
         with:
-          java-version: 1.11
+          java-version: 11
 
       - name: "Maven Test"
-        run: mvn clean test jacoco:report -DdryRun=true
+        run: mvn clean test jacoco:report -DdryRun=true
```

Comment:
Please add ending newline here and other files it is missing.

Reply:
New Line added